DATA/production/configurations/asyncReco/setenv_extra.sh

```diff
@@ -561,7 +566,13 @@ if [[ $ALIEN_JDL_EXTRACTTIMESERIES == 1 ]]; then
   if [[ ! -z "$ALIEN_JDL_ENABLEUNBINNEDTIMESERIES" ]]; then
     export ARGS_EXTRA_PROCESS_o2_tpc_time_series_workflow="$ARGS_EXTRA_PROCESS_o2_tpc_time_series_workflow --enable-unbinned-root-output --sample-unbinned-tsallis --threads 1"
   fi
-  if [[ ! -z "$ALIEN_JDL_SAMPLINGFACTORTIMESERIES" ]]; then
+  if [[ $ON_SKIMMED_DATA == 1]] || [[ ! -z "$ALIEN_JDL_SAMPLINGFACTORTIMESERIES" ]] ; then
+    if [[ $ON_SKIMMED_DATA == 1]] ; then
+      SAMPLINGFACTORTIMESERIES=0.00001f
```

Comment:
For skimmed production shoul be 0.1

Reply:
Sorry, you are definitely right, of course. I will fix it.

Reply:
Done, can you please recheck that everything is in order? @matthias-kleiner , @miranov25